src/v/v8_engine/tests/instance_test.cc

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2021 Vectorized, Inc.
+ *
+ * Licensed as a Redpanda Enterprise file under the Redpanda Community
+ * License (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * https://github.com/vectorizedio/redpanda/blob/master/licenses/rcl.md
+ */
+
+#include "seastarx.h"
+#include "v8_engine/environment.h"
+#include "v8_engine/instance.h"
+
+#include <seastar/testing/thread_test_case.hh>
+
+#include <boost/algorithm/string/case_conv.hpp>
+#include <boost/test/unit_test.hpp>
+
+#include <cstdint>
+
+const std::string_view to_upper_case_js = "function to_upper(obj) {\
+                                            let array = new Int8Array(obj);\
+                                                for (let i = 0; i < obj.byteLength; i++) {\
+                                                    if (array[i] >= 97 && array[i] <= 122) {\
+      	                                                array[i] = array[i] - 32;\
+                                                    } else {\
+                                                        array[i] = array[i];\
+                                                    }\
+                                                }\
+                                            }";
+
+const std::string_view sum_js = "function sum(obj) {\
```

Comment:
Nice tests! I would say maybe we can have a directory where you can place uncompiled wasm scripts for testing. Maybe in `src/v/coproc/tests/wasm`

Reply:
It is good idea. I want add test for wasm function

Reply:
Done